scania-digital-design-system/tegel
# feat(tooltip): update method for tooltip

**Describe pull-request**  
Update method for tooltop.

**Solving issue**  
Fixes: -

**How to test**  
1. Checkout the branch.
2. Try moving a tooltip target in runtime.
3. Call the update method
4. Make sure the tooltips position is updated.
